packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -158,8 +182,10 @@ const optionStyles = () => (base: TBase, state: TState) => {
     transition: `border-color ${designTokens.transitionStandard},
       background-color ${designTokens.transitionStandard},
       color ${designTokens.transitionStandard}`,
-    paddingLeft: designTokens.spacing20,
-    paddingRight: designTokens.spacing20,
+    paddingLeft: designTokens.paddingLeftForSelectInputOptions,
+    paddingRight: designTokens.paddingRightForSelectInputOptions,
+    paddingTop: `${designTokens.paddingTopForSelectInputOptions}`,
+    paddingBottom: `${designTokens.paddingBottomForSelectInputOptions}`,
```

Comment:
We don't need the template literal here:
```suggestion
    paddingTop: designTokens.paddingTopForSelectInputOptions,
    paddingBottom: designTokens.paddingBottomForSelectInputOptions,
```